### ISSUE TITLE
how can i query taks done today 

### ISSUE THREAD

**Issue by rmGFW** [YU000jp/logseq-default-queries-journals#1]
how can i query task finished today (may create this task not in today's journal)

**Comment by YU000jp** [comment]
Try this
```
  :default-queries {:journals [

                              {
                               :title ["☑️ Today DONE task"]
                               :query (and (between today tomorrow) (task DONE) )
                               :group-by-page? false
                               :breadcrumb-show? false
                               }

}
```

**Comment by rmGFW** [comment]
this only list done task which was create today, if task is create before today, and done today ,will not meet the condition

**Comment by YU000jp** [comment]
Certainly, that's correct. This issue is indeed equivalent to the one mentioned (https://github.com/DimitryDushkin/logseq-plugin-task-check-date/issues/10).
As of now, a solution to this problem has not been provided.
I have "Confirmation DONE task" plugin, and I believe it might be achievable by adding hidden properties for the query using that plugin. Add this to my TODOs.
